src/Management/src/EndpointCore/HeapDump/EndpointServiceCollectionExtensions.cs

```diff
@@ -42,13 +42,13 @@ public static void AddHeapDumpActuator(this IServiceCollection services, IConfig
                 services.AddActuatorManagementOptions(config);
                 services.AddHeapDumpActuatorServices(config);
 
-                if (Platform.IsWindows)
+                if (RuntimeInformation.FrameworkDescription.StartsWith(".NET Core", StringComparison.InvariantCultureIgnoreCase) && Platform.IsWindows)
```

Comment:
So, only use WindowsHeapDumper when it is .NET Core and on Windows? Otherwise use HeapDumper for all other cases?

Reply:
Maybe add a comment here to explain the different scenarios you are handling.

Reply:
Yes